src/h3/api/_api_template.py

```diff
@@ -223,28 +336,53 @@ def polyfill(geojson, res, geo_json_conformant=False):
         return _out_unordered(mv)
 
     def h3_is_pentagon(h):
-        """
-        a pentagon should still pass is_cell(), right?
-        :returns: boolean
+        """ Returns `True` if input is a valid H3 Cell which is
+        a pentagon.
+
+        todo: a pentagon should still pass is_cell(), right?
+
+        Returns
+        -------
+        bool
         """
         return _cy.is_pentagon(_in_scalar(h))
 
     def h3_get_base_cell(h):
-        """
-        :returns: boolean
+        """ Return the parenting cell of resolution `0`.
+
+        todo/question: overlap with the `get_parent()` function?
+
+        Returns
+        -------
+        bool
         """
         return _cy.get_base_cell(_in_scalar(h))
 
     def h3_indexes_are_neighbors(h1, h2):
-        """
-        :returns: boolean
+        """ Returns true if `h1` and `h2` are neighboring cells.
+
+        Returns
+        -------
+        bool
```

Comment:
note there's an inconsistency between using `bool` and `boolean` as the return type in the doc strings.

Reply:
Everything should be `bool` now, with the recent diffs.